test/Common/Plugin/AddPluginLinkStats/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(SOURCES AddPluginStats.cpp)
+
+if(NOT CYGWIN AND LLVM_ENABLE_PIC)
+  set(SHARED_LIB_SOURCES ${SOURCES})
+
+  set(bsl ${BUILD_SHARED_LIBS})
+
+  set(BUILD_SHARED_LIBS ON)
+
+  add_llvm_library(AddPluginStats ${SHARED_LIB_SOURCES} LINK_LIBS LW)
```

Comment:
Can we please house the test plugins artifacts in `lib/test` object directory instead?